iroha/src/queue.rs

```diff
@@ -5,27 +5,11 @@ use std::time::Duration;
 use crossbeam_queue::ArrayQueue;
```

Comment:
Hmm in the queue I think it is a wrong rebase, you just removed the changes from the previous PR

Reply:
In general I don't see a need to change the queue here in anyway, as we discussed I think you can just use version that was recently committed.